include/rman.h

```diff
@@ -0,0 +1,35 @@
+#ifndef _SYS_RMAN_H_
+#define _SYS_RMAN_H_
+
+#include "bus.h"
+
+typedef size_t rman_addr;
+
+typedef struct rman rman_t;
+typedef struct rman_block rman_block_t;
+
+struct rman_block {
```

Comment:
I'm convinced `resource` should play the role of `rman_block`... shouldn't that be the case?